optuna/integration/sklearn.py

```diff
@@ -563,6 +563,19 @@ def classes_(self) -> OneDimArrayLikeType:
 
         return self.best_estimator_.classes_
 
+    @property
+    def cv_results_(self) -> Dict[str, Any]:
+        """Return the dictionary has list of all trials.
+        Returns:
+            A list of results of all trials.
+        """
```

Comment:
Sorry for the delayed response. Could you update the docstrings? I don't think this attribute returns all trials (i.e., `list[FrozenTrial]`). In my understanding, it returns a list of CV results corresponding to all the trials.

Reply:
Thanks!